<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="RPIPL-NB-MGR050" name="com.rp.pw.test.cases.PWTC1" tests="2" failures="1" timestamp="4 Mar 2015 15:02:31 GMT" time="107.875" errors="0">
  <testcase name="pwTestScenario1" time="59.946" classname="com.rp.pw.test.cases.PWTC1"/>
  <testcase name="pwTestScenario1" time="47.929" classname="com.rp.pw.test.cases.PWTC1">
    <failure type="atu.testng.reports.exceptions.ATUReporterStepFailedException">
      <![CDATA[[ATU Custom Reporter Step Failed Exception]
	at atu.testng.reports.listeners.ATUReportsListener.onTestSuccess(Unknown Source)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1899)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:778)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- pwTestScenario1 -->
</testsuite> <!-- com.rp.pw.test.cases.PWTC1 -->
